### PR TITLE
Change SideNav style

### PR DESCRIPTION
**Pull request by huadream**
I changed the style of nav. I think it's more functional.
image

### CODE REVIEW

**Comment by bwsw** [comment]
@huadream we will check soon. Thanks.

**Comment by bwsw** [comment]
@huadream we check on Monday. Thanks.

File: proxy-conf-example.js [hunk 1]
```diff
@@ -1,56 +0,0 @@
-const CLOUDSTACK_ENDPOINT = 'https://example.com';
```
Comment (dron8552) [inline proxy-conf-example.js]: why it was removed?

File: src/i18n/ru.json [hunk 2]
```diff
@@ -342,6 +342,7 @@
   "NAVIGATION_SIDEBAR": {
     "WELCOME": "Здравствуйте, {{ name }}",
     "VMS": "Виртуальные машины",
+    "CLOUD": "облако",
```
Comment (dron8552) [inline src/i18n/ru.json]: need to capitalize the first letter "Облако"

File: proxy-conf-example.js (hunk 1 above)
Comment (huadream) [reply proxy-conf-example.js]: Sorry for committing this file accidentally.

**Comment by huadream** [comment]
There are some codes useless but didn't clean because I am developing a topbar and responsive layout.

The sidenav will change to over mode.
image

The topbar has the nav menu and moves the search box to the topbar.
image

The menu button could become a back button and show the layout for mobile.
image

**Comment by huadream** [comment]
have merged to #1672

**Pull request closed without merging**